### PR TITLE
Implementation of MAX_POOL_2D operation

### PR DESCRIPTION
**Pull request by lsc31**
Signed-off-by: Lakshmishree C <Lakshmishree C lakshmishree.c@intel.com>

### CODE REVIEW

**Review by Nagamani71 — APPROVED** [review]
LGTM

**Review by akodanka — COMMENTED** [review]
Implementation looks good. A couple of minor suggestions though.

File: ngraph_creator/operations/src/Max_Pool_2d.cpp [hunk 1]
```diff
@@ -0,0 +1,205 @@
...
+        activationFn = sModelInfo->ParseOperationInput<uint32_t>(mNnapiOperationIndex, 9);
+
+        if (inputsSize == 11) {
+            layout = sModelInfo->ParseOperationInput<uint8_t>(mNnapiOperationIndex, 10);
+        }
+
+        if (layout) useNchw = true;
+
+        auto_pad = ngraph::op::PadType::EXPLICIT;
+        {
```
[the remark this answers is not in the document]
Comment (lsc31) [reply ngraph_creator/operations/src/Max_Pool_2d.cpp]: Fixed.

**Review by akodanka — APPROVED** [review]
LGTM

**Pull request merged**